commons-app/pom.xml

```diff
@@ -282,6 +282,18 @@
 			<version>${mysql.jdbc.version}</version>
 		</dependency>
 
+		<dependency>
```

Comment:
We need this to support mariadb (mysql fork at recent versions of fedora) and to support MS SQL server (jdbc driver which is currently hosted at SWCs nexus repository because it is, in best Microsoft tradition, not available under a public license)